### DISCUSSION TITLE
Do we need a premium real-debrid

### DISCUSSION THREAD

**Discussion by magic777** [itsToggle/plex_debrid#351] [Q&A]
do we need a premium real-debrid account coz after running plex_debrid there is an error? 

" [realdebrid] error: (403 unauthorized): You may have attempted to add an infringing torrent or your realdebrid account is locked or you dont have premium. "

**Comment by lstryhrn** [comment]
Yes because your api account doesnt allow that typle of connection. I didnt even think it was a free version. If you are testing the waters just sign up for a month.  Its dirt cheap
